content/en/blog/_posts/2024-06-28-cpumanager-new-static-policy.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: 'Kubernetes v1.30: CPUManager new static policy-XXXX'
+date: 2024-06-28
```

Comment:
@Jeffwan your publication date is 2024-08-22 instead of the current one, can you update it?

Reply:
updated